docs/verifiable-document/document-template.md

```diff
@@ -4,8 +4,285 @@ title: Deploying Document Renderer
 
-TBD
+## Pre-requisite
+
+- [git](https://git-scm.com/downloads)
+- [node.js](https://nodejs.org/en/download/)
+- [Github account](https://github.com/)
+- [Netlify account](https://www.netlify.com/)
+- [Basic react.js knowledge](https://reactjs.org/)
+
+## Clone Decentralized Renderer React Template
+
+A template for building your own document renderer has been created for you at https://github.com/Open-Attestation/decentralized-renderer-react-template.
+
+### Clone code repository locally
+
+```sh
+git clone https://github.com/Open-Attestation/decentralized-renderer-react-template.git
+```
+
+### Install code dependencies
+
+```sh
+cd decentralized-renderer-react-template
+npm install
+```
+
+### Run development preview
+
+```sh
+npm run storybook
+```
+
+### Development environment
+
+![Default Story Book View](/docs/verifiable-document/document-template/default-storybook.png)
+
+After running the storybook, you should be able to see the default template provided at http://localhost:6006/.
+
+This is a live preview where you can:
+
+1. edit the raw document data in the "Knobs" tab
+1. edit the template code to render the data
+
+## Creating raw document data
+
+In the "Knobs" tab of storybook, replace the document data with the following:
+
+```json
+{
+  "$template": {
+    "name": "COC",
+    "type": "EMBEDDED_RENDERER",
+    "url": "http://localhost:3000"
+  },
+  "name": "OpenAttestation Tutorial Certificate of Completion",
+  "recipient": {
+    "name": "John Doe"
+  }
+}
+```
+
+In the above document data, you see three root objects:
+
+### 1. \$template
+
+The `$template` key to describe the template name used to render this display.
+
+`$template.type` will always take the value of `EMBEDDED_RENDERER` for documents rendered in this manner.
+
+`$template.name` is the name of the template used to render a given OA document. This allows a single document renderer to render for multiple types of OA documents; each with a different template name.
+
+### 2. name
+
+The `name` key is a compulsory key to describe the type of OA document. In this case, we are creating a `OpenAttestation Tutorial Certificate of Completion`.
+
+### 3. recipient
+
+OA documents do not have strict data structure and allows issuers of documents to define their own data schema. The `recipient` object is a user-defined object that describes who the certificate is conferred to. In this case, you may replace `John Doe` with your name.
+
+In the next section, you will learn more about the OA document schema and how you may defined your own data structure, but for now, we will stick to this simple document data.
+
+## Developing the renderer
+
+### Copying the provided template
+
+```sh
+cp -r src/templates/customTemplate src/templates/coc
+```
+
+Copy the entire "customTemplate" folder in `src/templates/customTemplate` and renamed it as "coc" in `src/templates/coc`.
+
+This will create an unique template just to render `COC` templates later.
```

Comment:
an unique => a unique (https://www.quora.com/Which-is-the-correct-grammar-usage-a-unique-or-an-unique)

Reply:
WHAT!?!?!?!

Reply:
an unique sounds weird to my ears, that's why I checked haha :D